src/sklearn_evaluation/plot/roc.py

```diff
@@ -196,15 +196,15 @@ def _set_ax_settings(ax):
     ax.set_xlabel("False Positive Rate")
     ax.set_ylabel("True Positive Rate")
     ax.set_title("ROC")
-    ax.legend(loc="best")
+    ax.legend(loc="best", fontsize=8)
```

Comment:
setting fontsizes is tricky because this is a relative measure. 8 points might look good in some resolutions, but not others, so we don't set these values.

let's remove the fontsize so we can merge the linestyle part, then open another PR to deal with the legend location part. I guess nothing worked by changing the loc parameter? I'd expect it to have some value to ensure it'd be outside the plot, if that isn't the case, we can manually locate it outside, we need to experiment a bit

Reply:
Okay that makes sense, I should've thought of that.

The loc parameter only accepts values like `'center right'` or `'upper left'` which only changed the position within the plot. The way I've gathered you get it outside of the plot is by altering a parameter `bbox_anchor`, which produced inconsistent results depending on the plot for some reason. I was trying different values for a while and some values worked well for certain plots but oddly placed the legend in others.

Reply:
ok, let's remove this for now then!